### ISSUE TITLE
Possibility to remove obsolete Account Information

### ISSUE THREAD

**Issue by J0000** [Project60/org.project60.sepa#422]
Sometimes clients change their IBAN. You can create a mandate with a new IBAN but you are not able to remove the old number. 

It would be nice to have the possibility to remove the old bank information, perhaps you cloud be asked for that when you create a SEPA mandate with a new IBAN?

**Comment by systopia** [comment]
Good idea, @J0000. However, it should not only be possible to 'retire' an IBAN when creating a new mandate. Any better ideas?

**Comment by J0000** [comment]
No, not only, but it's a good opportunity :) 

A button with a small Pop-Up in the Contribution tab? Or anywhere in connection with CiviBanking?
